Core/include/Acts/Propagator/RiddersPropagator.ipp

```diff
@@ -17,42 +17,31 @@ auto Acts::RiddersPropagator<propagator_t>::propagate(
       action_list_t_result_t<CurvilinearTrackParameters,
                              typename propagator_options_t::action_list_type>>;
 
+  auto startWithoutCov = start;
```

Comment:
Explicit type would be preferable here I think, plus I usually try to add a comment to indicate the copy is intentional just to be safe.